internal/github/user.go

```diff
@@ -28,17 +27,16 @@ func NewUserHelper(gc *github.Client, repoName *RepoName) UserHelper {
 	}
 }
 
-var ErrUnexpectedReply = errors.New("the users search result are incoclusive")
+var ErrUnexpectedReply = errors.New("the number of found commits isn't exactly 1")
 
-func (uh *UserHelperImpl) GetUser(ctx context.Context, commit *object.Commit) (*github.User, error) {
+func (uh *UserHelperImpl) GetUser(ctx context.Context, sha string) (*github.User, error) {
```

Comment:
```suggestion
func (uh *UserHelperImpl) GetCommitAuthorctx context.Context, sha string) (*github.User, error) {
```

Reply:
I am not sure about this one. This is not a commit author. It is a Github `User`, especially the Github's user `Login` that we are interested in.

Reply:
The function signature makes that explicit enough IMO.

Reply:
Fixed.